pkg/systemd/services/services-list.jsx

```diff
@@ -40,38 +40,39 @@ export const ServicesList = ({ units, isTimer }) => {
     );
 };
 
-const ServicesRow = ({ Id, shortId, AutomaticStartup, UnitFileState, LoadState, HasFailed, CombinedState, LastTriggerTime, NextRunTime, Description, WantedBy, isTimer }) => {
-    const props = { shortId, Description };
-    const columnsMap = {
-        shortId: { value: _("Name"), className: "service-unit-id", width: 2 },
-        Description: { value: _("Description"), className: "service-unit-description", width: 4 },
-        Triggers: { value: _("Triggers"), timerOnly: true, className: "service-unit-triggers", width: 3 },
-    };
+class ServicesRow extends React.PureComponent {
```

Comment:
Feels like a step backwards to have to convert from a simple function to a class -- but apparently [pure functional components](https://blog.logrocket.com/pure-functional-components-in-react-16-6/) need another npm module..

Reply:
I can drop that commit if you don't prefer it - I just wanted to try it out basically.
I don't want to bring in a new NPM module just for this.

Reply:
Sorry, that wasn't meant as a veto, more like a "sigh" :-)

Reply:
I don't think we should have the rule to use functions if at all possible. Always using a class is more uniform and we don't have to flip back and forth between using a function and using a class depending on whether a component has state or not, or derives from PureComponent or not, or needs some lifecycle methods or not.